torch/distributions/transformed_distribution.py

```diff
@@ -57,26 +57,29 @@ def __init__(self, base_distribution, transforms, validate_args=None):
         base_shape = base_distribution.batch_shape + base_distribution.event_shape
         base_event_dim = len(base_distribution.event_shape)
         transform = ComposeTransform(self.transforms)
-        domain_event_dim = transform.domain.event_dim
-        if len(base_shape) < domain_event_dim:
+        if len(base_shape) < transform.domain.event_dim:
             raise ValueError("base_distribution needs to have shape with size at least {}, but got {}."
-                             .format(domain_event_dim, base_shape))
-        shape = transform.forward_shape(base_shape)
-        expanded_base_shape = transform.inverse_shape(shape)
+                             .format(transform.domain.event_dim, base_shape))
+        sample_shape = transform.forward_shape(base_shape)
```

Comment:
`sample_shape` has precise meaning in `torch.distributions`, and that meaning differs from the proposed use here.  Is there another variable name you could think of using?

Reply:
How about `forward_shape`?
I pushed that change, but please feel also free to just close without merging if you feel like it's not worth the trouble.